ci/run-check.sh

```diff
@@ -10,11 +10,11 @@ cargo --version
 
 case $TARGET in
   cargo-build)
-    cargo build --release "$@"
+    cargo build -p centrifuge-chain --release "$@"
     ;;
 
   test-general)
-    cargo test --workspace --release --features runtime-benchmarks,try-runtime --exclude runtime-integration-tests
+    cargo test --release --features runtime-benchmarks,try-runtime,fast-runtime --exclude runtime-integration-tests
```

Comment:
If we compile with `fast-runtime` here, it "ideally" should reuse the compiled runtimes later for testing

Reply:
I reject myself here. The cache is gathered from the previous merge in the same CI job, no across jobs in the same PR.